_ml-commons-plugin/api/agent-apis/register-agent.md

```diff
@@ -26,12 +34,17 @@ POST /_plugins/_ml/agents/_register
 
 The following table lists the available request fields.
 
-Field | Data type | Required/Optional | Description
-:---  | :--- | :--- 
-`name`| String | Required | The agent name. |
-`type` | String | Required | The agent type. Valid values are `flow` and `conversational`. For more information, see [Agents]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/index/). |
-`description` | String | Optional| The agent description. |
-`tools` | Array | Optional | A list of tools for the agent to execute. 
+Field | Data type | Required/Optional | Agent type | Description
+:---  | :--- | :--- | :--- | :---
+`name`| String | Required | All | The agent name. |
+`type` | String | Required | All | The agent type. Valid values are `flow` and `conversational`. For more information, see [Agents]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/index/). |
+`description` | String | Optional| All | The agent description. |
+`tools` | Array | Optional | All | A list of tools for the agent to execute. 
+`app_type` | String | Optional | All | Specifies an optional agent category. You can then perform operations on all agents in the category. For example, you can delete all messages for RAG agents.
+`memory.type` | String | Optional | `conversational_flow`, `conversational` | Specifies where to store the conversational memory. Currently, the only supported type is `conversation_index` (store the memory in a conversational system index).
+`llm.model_id` | String | Required | `conversational` | The model ID of the LLM to which to send questions.
+`llm.parameters.response_filter` | String | Required | `conversational` | The pattern for parsing the LLM response. For each LLM, you need to provide the field where the response is located. For example, for the Anthropic Claude model, the response is in the `completion` field so the pattern is `$.completion`. For OpenAI models, the pattern is `$.choices[0].message.content`.
```

Comment:
"in which the response is located." "the response is located in the `completion` field, so" (add "located" and a comma after "field").